internal/tsc_wrapped/compiler_host.ts

```diff
@@ -343,10 +343,8 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
    * file reads.
    */
   fileExists(filePath: string): boolean {
-    // Allow moduleResolution=node to behave normally.
-    if (this.allowNonHermeticReads && isNonHermeticInput(filePath) &&
-        this.delegate.fileExists(filePath)) {
-      return true;
+    if (this.allowActionInputReads) {
```

Comment:
add docs here? maybe actually expand to document the pros and cons of what we discussed? It'll be hard to swap that discussion back into your brain in three months.

Reply:
done